onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -78,6 +78,12 @@ OnnxRuntimeTestSession::OnnxRuntimeTestSession(Ort::Env& env, std::random_device
     Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_DML(session_options, 0));
 #else
     ORT_THROW("DirectML is not supported in this build\n");
+#endif
+  } else if (provider_name == onnxruntime::kAclExecutionProvider) {
+#ifdef USE_ACL
+    Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_ACL(session_options, 0));
```

Comment:
>0 [](start = 84, length = 1)

replace with performance_test_config.run_config.enable_cpu_mem_arena ? 1 : 0

Reply:
Thanks!

Reply:
Fixed in v3